src/components/Alert/Alert.tsx

```diff
@@ -16,7 +21,9 @@ const Alert: React.FC<AlertProps> = ({ onClose, ...rest }) => {
     }
   }, [setOpen, onClose]);
 
-  return (
+  return disableAnimation ? (
+    <ControlledAlert {...rest} open={open} onClose={handleClose} />
+  ) : (
```

Comment:
`[stylistic, non-blocker]` personally, since the condition is at the top-level and simple, I'd suggest we go with an `if` block.

```
if (disableAnimation) {
  return <ControlledAlert {...rest} open={open} onClose={handleClose} />
}

return <Collapse> // etc
```

Not a blocker though!

Reply:
Done! I've also included the test and examples to the MDX file.